package.json

```diff
@@ -19,6 +19,7 @@
         "typescript": "~3.7.2"
     },
     "devDependencies": {
+        "mustache-express": "^1.3.0",
```

Comment:
Mustache brukes til å erstatte NAV_SCRIPTS, NAV_STYLES osv. i HTML-filen. Mustache-express er for at express skal klare å gjøre dette med index-fila når den serves.